### ISSUE TITLE
Attention Mask 생성 관련 질문

### ISSUE THREAD

**Issue by Baekyeongmin** [joongbo/tta#4]
안녕하세요! 좋은 논문과 코드 공개해주셔서 감사합니다!

코드를 읽던 중 궁금한게 생겨서 이슈 남기게 되었습니다.

```python
attention_mask = create_attention_mask_from_input_mask(input_ids, input_mask)
attention_mask = 1.0 - tf.linalg.band_part(attention_mask, 0, 0) ## for self-blind
```

이 부분에서 제가 이해한 것이 맞다면, `input_mask`는 실제 토큰(1)과 패딩값(0)을 분리하는 Mask이고, 첫 번째 줄은 `input_id`와 `input_mask`로부터 [`batch_size`, `from_sequence_length`, `to_sequence_length`] 형태로 만듭니다.
그리고 두 번째줄은 토큰 자기 자신을 Attend하는 것을 막기 위해 대각선 값들을 0으로 바꿔주는 것으로 이해했습니다. 하지만 1.0에서 `tf.linalg.band_part(attention_mask, 0, 0)` 값을 빼면, `input_mask`에 담겨있던 패딩에 대한 정보가 없어지는 것 같이 보입니다. 그래서 두 번째 줄을 아래와 같이 바꿔야 하는 것이 아닌지 궁금합니다.

```python
attention_mask = attention_mask - tf.linalg.band_part(attention_mask, 0, 0) ## for self-blind
```

감사합니다!

**Comment by joongbo** [comment]
질문자께서 말씀하신 부분이 맞습니다.

업데이트 하는 걸 깜박하고 있었습니다. 바로 수정했습니다. 혼란을 드려 죄송합니다. 

상기시켜주셔서 감사합니다.

**Comment by Baekyeongmin** [comment]
확인 감사합니다!
